hummingbot/market/celo/celo_cli.py

```diff
@@ -0,0 +1,57 @@
+import subprocess
+from subprocess import CalledProcessError
+from decimal import Decimal
+
+
+symbols_map = {"CGLD": "gold", "CUSD": "usd"}
+
+
+def command(command_text: str):
+    try:
+        output = subprocess.check_output(command_text, stderr=subprocess.STDOUT, shell=True)
```

Comment:
I'd recommend `shell=False` instead here, and pass in the arguments with a list. As it is written right now... any shell escape characters in the password would cause the `CeloCli` class to break.